src/Orleans.GrainDirectory.Redis/Options/RedisGrainDirectoryOptions.cs

```diff
@@ -21,6 +39,10 @@ public RedisGrainDirectoryOptionsValidator(RedisGrainDirectoryOptions options)
 
         public void ValidateConfiguration()
         {
+            if (this.options == null)
```

Comment:
Can this ever be null? We should be using IOptions<T>, not just T, right?

Then, we can check the `.ConfigurationOptions` isn't null, instead